src/PrestaShopBundle/Twig/TranslationsExtension.php

```diff
@@ -434,12 +535,24 @@ protected function makeSubdomainPrefix($level)
      */
     protected function tagSubject($subject, $level, $id = null)
     {
-        $openingTag = '';
-        $closingTag = ' ';
+        $openingTag = '<h2 class="domain-part"><i class="material-icons delegate-toggle-messages">&#xE315;</i>'.
+            '<span class="delegate-toggle-messages{{ missing translations class }}">';
+        $closingTag = '</span>{{ missing translations warning }}</h2>';
 
         if (2 === $level) {
-            $openingTag = '<h2>';
-            $closingTag = '</h2>';
+            $openingTag = '<h2 class="domain-first-part"><i class="material-icons">&#xE315;</i><span>';
+            $closingTag = '</span></h2>'.
+                '<div class="domain-actions">' .
+                    '<i class="material-icons visibility-on">&#xE8F4;</i>'.
+                    '<i class="material-icons visibility-off hide">&#xE8F5;</i>'.
+                    '<span class="btn-show-messages">' .
+                        $this->translator->trans('Show messages', array(), 'Admin.International.Feature') .
```

Comment:
I meant this "Hide messages" - I couldn't see it, only "Show messages"

![image](https://cloud.githubusercontent.com/assets/7910738/19555323/a32925e6-96bc-11e6-9334-afd6cb0eba6c.png)

Thanks!